app/Cardano/Shell/Features/Logging.hs

```diff
@@ -62,9 +61,8 @@ createLoggingFeature cardanoEnvironment cardanoConfiguration = do
     -- we parse any additional configuration if there is any
     -- We don't know where the user wants to fetch the additional configuration from, it could be from
     -- the filesystem, so we give him the most flexible/powerful context, @IO@.
-    loggingConfiguration    <-  pure $ LoggingParameters
-                                    (ccLogConfig cardanoConfiguration)
-                                    (ccLogPath cardanoConfiguration)
+    loggingConfiguration    <-  pure $ LoggingParameters $
```

Comment:
The idea is that you _load your configuration_ here, not pass the name of the configuration that needs to be loaded on a function that then loads the configuration. Reduce indirection.